dashboard/app/models/script_level.rb

```diff
@@ -21,10 +21,13 @@
 #  index_script_levels_on_stage_id   (stage_id)
 #
 
+require 'cdo/shared_constants'
+
 # Joins a Script to a Level
 # A Script has one or more Levels, and a Level can belong to one or more Scripts
 class ScriptLevel < ActiveRecord::Base
   include LevelsHelper
+  include SharedConstants
```

Comment:
Yes, that's what I expect. If you want to use this within a `ScriptLevel` class instance, then include will be sufficient since it applies all module members to the instance.